setup.cfg

```diff
@@ -0,0 +1,9 @@
+[flake8]
```

Comment:
see also https://github.com/apache/spark/blob/master/conf/fairscheduler.xml.template, if it is possible, we need a license header here also

Reply:
Done